src/math/vec.zig

```diff
@@ -910,3 +917,30 @@ test "Mat4x4_mulMat" {
     const expected = math.vec4(7, 9, 6, -1);
     try testing.expect(math.Vec4, expected).eql(m);
 }
+
+test "Vec2_fromInt" {
+    const x: i8 = 0;
+    const y: i32 = 1;
+    const v = math.Vec2.fromInt(x, y);
```

Comment:
Just like how we have `math.vec2(x, y)` which is a helper/alias for `math.Vec2.init(x, y)`... could you add a helper/alias for `math.vec2FromInt(x, y)` and use it in these tests instead? You'd write that here: https://github.com/hexops/mach/blob/fc4c3d06a306f49f01db347e6b1fb8561fb10c72/src/math/main.zig#L76-L98

Reply:
Oh Yeah. Will add those in main.zig